test/fixtures/env.js

```diff
@@ -1,4 +1,11 @@
 "use strict";
 
+global.log = {
+	error: () => console.error.apply(console, arguments),
+	warn: () => {},
+	info: () => {},
+	debug: () => {},
+};
+
```

Comment:
I've seen this is usually done via a `LOG_LEVEL` env var or the like, but I'm fine either way.